wurst/systems/LumberAndCoinSystem.wurst

```diff
@@ -15,6 +15,28 @@ var coinCount = GetRandomInt(-2, 3)
 var coinCap = 16
 var coinsSpawned = 0
 
+/**
+	Add `amt` lumber to each player's pool.  If `redistribute`, players with too
+	much lumber will be forced to share.
+*/
+public function LinkedList<PlayerData>.addLumber(int amt, bool redistribute)
+	for pd in this
+		pd.p.addLumber(amt)
+
+	if redistribute and this.size() > 1
+		for pd in this
+			if pd.p.getLumber() > WOOD_REDISTRIBUTE_THRESHOLD
+				let val = (pd.p.getLumber() / allPlayers.size()).toInt()
+				let remainder = pd.p.getLumber() mod allPlayers.size()
+				pd.p.setLumber(remainder)
+				allPlayers.addLumber(val, false)
+				printTimedToPlayer(
+					"Due to inactivity, your lumber has been dispersed amongst your teammates!",
+					10.,
+					pd.p
+				)
```

Comment:
fix formatting

Reply:
@Frotty what do you mean "fix"? You want this all on one long line?

Reply:
no floating bracket